7_deploy_osh/roles/suse-osh-deploy/templates/suse-horizon.yaml.j2

```diff
@@ -0,0 +1,2 @@
+---
+duMMyKey: duMMyValue
```

Comment:
I am not sure I am following this.

Reply:
This is part of the values override for installing the chart but I'm not sure where it gets consumed either. Comment here would be super helpful.

Reply:
There seems to be bug in ansible config_template. When src template file is empty, I was seeing error as mentioned in rocket chat discussion ( https://chat.suse.de/channel/cloud-ccp?msg=ar8HekzS2TwHR9zdj ).
I am kind of surprised this is happening as its an upstream openstack ansible module . Ideally it needs to be addressed in config_template ansible module.

To get around this issue, I am just adding a dummy entry so that it does not match and override any chart value. After adding this entry, I don't see that error.